Sierra/challenge_07/box_office_challenge.py

```diff
@@ -0,0 +1,43 @@
+# this variable is assigned a list of strings
+top_50_list = ['Star Wars: Episode I - The Phantom Menace', 'The Sixth Sense', 'Austin Powers: The Spy Who Shagged Me', 'Toy Story 2', 'The Matrix', 'Tarzan', 'Big Daddy', 'The Mummy', 'Runaway Bride', 'The Blair Witch Project', 'Notting Hill', 'The World Is Not Enough', 'Double Jeopardy', 'Wild Wild West', 'Analyze This', "The General's Daughter", 'American Pie', 'Inspector Gadget', 'Shakespeare in Love', 'Sleepy Hollow', 'The Haunting', 'Patch Adams', 'Entrapment', 'Pokémon: The First Movie - Mewtwo Strikes Back', 'Payback', 'Deep Blue Sea', 'American Beauty', 'The Thomas Crown Affair', 'Stuart Little', 'Blue Streak', 'The Green Mile', 'Bowfinger', 'Life', 'The Bone Collector', "She's All That", 'End of Days', 'Three Kings', 'A Civil Action', 'Stepmom', 'Eyes Wide Shut', 'Never Been Kissed', 'Forces of Nature', 'Varsity Blues', 'Message in a Bottle', "You've Got Mail", 'South Park: Bigger, Longer & Uncut', 'Stigmata', 'Life Is Beautiful', 'The Prince of Egypt',
+               'Deuce Bigalow: Male Gigolo']
+
+
+# TODO: Take the string "movies_str" and convert it into a list and assign it to a variable called "remaining_50_list"
+# Note: Print your new variable to see the list of movies
+remaining_50_list = movies_str.split(',')
+print(remaining_50_list)
+
+# TODO: Hmm. Looks like "remaining_50_list" contains the movie "The Matrix" which is also in the "top_50_list". Let's remove this duplicate title from "remaining_50_list".
+# Note: Make sure to double check the list you are removing from!
+remaining_50_list.remove('The Matrix')
+print(remaining_50_list)
+
+
+# TODO: We're down to 49 movies in "remaining_50_list". Let's add a movie to the list called "Anywhere But Here"
+# Note: Print the variable "remaining_50_list" and see if the movie has been added!
+remaining_50_list.append('Anywhere But Here')
+print(remaining_50_list)
+
+
+# TODO: Let's make sure "remaining_50_list" has exactly 50 movie titles
+print(len(remaining_50_list))
+
+# TODO: Great! Let's join the two lists into one list called "top_100". Uncomment the line of code below and run your code to see how it's done!
+# Note: Print your new variable to see the combined list of movies
+top_100 = remaining_50_list + top_50_list
+print(top_100)
+
+# TODO: Let's print the first movie in our "top_100" list. Let's use the index lookup method to do this.
+print(top_100[0])
+
+# TODO: Let's print the top 10 movies in "top_100" as a list. Let's use the slice syntax to accomplish this!
+print(top_100[0:10])
```

Comment:
This will actually print out the top 11 movies instead of top 10. Remember, 0 counts as the first, so top ten should be from 0 to 9.

Reply:
When I do "print(top_100[0:9])" it only prints from index 0 to index 8 and produces the top 9. When I do "print(top_100[0:10])" it prints from index 0 to index 9 and produces the top 10. Hopefully the images show up I tried to add them to show you what I mean. 

![slice 0_9](https://user-images.githubusercontent.com/85408798/154014338-d4b62faa-137a-4d87-a458-6a18407572c5.png)
![slice 0_10](https://user-images.githubusercontent.com/85408798/154014341-3c3f05a0-4e43-4ac5-89e6-eff750dff7af.png)

Reply:
Thanks for the clarification. I guess it's a python3 change. Sorry I messed it up.

Reply:
It's okay, thank you!